### DISCUSSION TITLE
libre 2 web follower interval problem.

### DISCUSSION THREAD

**Discussion by iicd2386cd** [NightscoutFoundation/xDrip#3353] [Q&A]
Hi, I use libre 2 in combination with xdrip which downloads data via web fallower. Everything works, but the problem is that it collects data every 10 minutes instead of every 5, I've tried everything,Any ideas what might be wrong?

**Comment by PhilTU** [comment]
This was also reported on Facebook. It is possible that Abbott have changed something on the server end.

To increase visibility of logging add xtra tags for logging : WebFollower:v,WebFollow:v
in settings of xDrip+ (search settings for "logging").

might give you some ideas

**Reply by iicd2386cd** [reply]
Can you explain to me how to do this?

**Reply by iicd2386cd** [reply]
Can you explain to me how to do this?

**Reply by Navid200** [reply]
Go to Settings &#8722;> Less common settings &#8722;> Extra Logging Settings &#8722;> Extra tags for logging

Therefore, you can enter the name of the java routines that you want to get more logs from followed by :v.

**Reply by iicd2386cd** [reply]
I did it but it doesn't mean anything to me 🫠 would you like to take a look at it?

**Reply by Navid200** [reply]
The logs may contain private info.
So, blackout any such info like passwords or IDs before posting.

**Comment by PhilTU** [comment]
> I did it but it doesn't mean anything to me 🫠 would you like to take a look at it?

Posting it here is the best chance someone who does understand it will see it, so post away!

Also if you want to add philtuk at outlook.com as a Librelinkup follower I can see if my spare phone does the same. You can remove me later.
